src/djangooidc/views.py

```diff
@@ -109,12 +109,10 @@ def logout(request, next_page=None):
         # Don't wait for the callback as it may never come.
         auth_logout(request)
         logger.info("Successfully logged out user %s" % username)
-        next_page = getattr(settings, "LOGOUT_REDIRECT_URL", None)
-        if next_page:
-            request.session["next"] = next_page
+        return HttpResponseRedirect(redirect_to="https://get.gov")
```

Comment:
The subtle behavior here is that a return statement in a finally block will preempt a return statement in the try clause. In other words, this will replace `return HttpResponseRedirect(url)` with `return HttpResponseRedirect(redirect_to="https://get.gov")` (skipping Login.gov entirely).

If you'd like folks to end up at "https://get.gov" after logging out, there's no real need for them to even return to the Django site at all. Ensure that the partner dashboard of Login.gov is configured to allow get.gov as a post-logout redirect destination and then send get.gov as the value of the GET parameter `next` in the logout request.

The prior code ought to work, if you set `LOGOUT_REDIRECT_URL` to what you'd like, although I haven't tested this.